packages/nc-gui/components/dashboard/TreeView/ProjectNode.vue

```diff
@@ -267,7 +272,9 @@ const onProjectClick = async (project: NcProject, ignoreNavigation?: boolean, to
 }
 
 function openErdView(base: BaseType) {
-  navigateTo(`/nc/${base.project_id}/erd/${base.id}`)
+  if (!base?.id) return
```

Comment:
`id` is always available in `base` and base won't be `null` here since you've checked in L290. so this line can be removed.